src/components/app/Scores/index.tsx

```diff
@@ -1,29 +1,130 @@
-import React, { useState } from 'react';
-import { Typography } from '@material-ui/core';
+/* eslint-disable react/display-name */
+import { useState } from 'react';
+import { Tooltip, Typography } from '@material-ui/core';
+
 import { useOnGet } from '@typesaurus/react';
-import { useAuth } from '../../../contexts/AuthContext';
 import { init as initFB } from '../../../utils/firebase';
 import { Grouping } from '../../../types/grouping';
+import { GenericList, GenericListItem } from '../GenericList';
+import { HeadCell } from '../GenericList/HeadCell';
+import _ from 'lodash';
+import { getAverage, getMedian } from '../../../utils/commonUtils';
+import { ScoreRowCollapse } from '../../common/ScoreRowCollapse';
+import { Semaphore } from '../../common/Semaphore';
+import { Link } from 'react-router-dom';
+import MonetizationOnRoundedIcon from '@material-ui/icons/MonetizationOnRounded';
 
-// TODO: complete this component
+const headCells: HeadCell[] = [
+    {
+        id: 'averageScore',
+        numeric: false,
+        disablePadding: false,
+        label: 'Score',
+        align: 'center',
+        format: (item: GenericListItem, value: string | number | boolean) => {
+            return <Semaphore value={value} />;
+        },
+    },
+    {
+        numeric: false,
+        disablePadding: false,
+        label: 'Actions',
```

Comment:
why is this called actions?

Reply:
Yes, the initial idea was to have more links to different pages. Maybe it can be renamed to Links?

Reply:
TVL Link seems to be the name at least from what is showing